src/Database/PostgreSQL/Consumers/Config.hs

```diff
@@ -83,19 +83,27 @@ data ConsumerConfig m idx job = forall row. FromRow row => ConsumerConfig {
 , ccJobFetcher            :: !(row -> job)
 -- | Selector for taking out job ID from the job object.
 , ccJobIndex              :: !(job -> idx)
--- | Notification channel used for listening for incoming jobs. If set
--- to 'Nothing', no listening is performed and jobs are selected from
--- the database every 'ccNotificationTimeout' microseconds.
+-- | Notification channel used for listening for incoming jobs.
+-- Whenever the consumer receives a notification, it checks the database
+-- for any pending jobs ('run_at <= NOW()') and runs them all.
+-- If set to 'Nothing', no listening is performed and 'ccNotificationTimeout'
+-- should be set to positive number, otherwise no jobs would be ever run.
+-- 'ccNotificationChannel' and 'ccNotificationTimeout' can be combined.
+-- Pending jobs will be run either when notification is received or no
```

Comment:
Read the whole thread please. There are some important details that basically require altering the description a bit.
https://scrive.slack.com/archives/CRCQ3T537/p1637320243443300?thread_ts=1637310773.420100&cid=CRCQ3T537

Last sentence should be something like this.

> Consumer will check for pending jobs either when notification is received or no notification is received for 'ccNotificationTimeout' microseconds since the last check.

Reply:
Done.